### PR TITLE
[PHP] Do not highlight plain SQL indicator as SQL

### PR DESCRIPTION
**Pull request by jfcherng**
```php
    $sql = "SELECT";
    //      ^^^^^^ - keyword.other.DML
```

Fixes https://github.com/sublimehq/Packages/issues/2956

### CODE REVIEW

**Review by keith-hall — APPROVED** [review]

**Review by Thom1729 — DISMISSED** [review]

File: PHP/PHP Source.sublime-syntax [hunk 1]
```diff
@@ -9,7 +9,7 @@ variables:
   identifier: '\b{{identifier_start}}[[:alnum:]_]*\b'
   path: '\\?({{identifier}}\\)*{{identifier}}'
   regex_modifier: '[eimsuxADJSUX]*'
-  sql_indicator: \s*(?:SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER)\b
+  sql_indicator: \s*(?:SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER)(?=$|\s)
```
Comment (deathaxe) [inline PHP/PHP Source.sublime-syntax:12]: I don't think we need to highlight `SELECT` as keyword in an EOF situation, if we don't do so in situations of it being the only keyword in a quoted string.

**Comment by deathaxe** [comment]
I wonder how long it takes until someone complains about `"SELECT" . cols . "FROM"` not being correctly highlighted.

File: PHP/PHP Source.sublime-syntax (hunk 1 above)
Comment (jfcherng) [reply PHP/PHP Source.sublime-syntax:12]: @deathaxe that would fail one of added tests.

**Comment by jfcherng** [comment]
> I wonder how long it takes until someone complains about `"SELECT" . cols . "FROM"` not being correctly highlighted.

It has never been done iirc. And unlikely will be done.

File: PHP/PHP Source.sublime-syntax (hunk 1 above)
Comment (deathaxe) [reply PHP/PHP Source.sublime-syntax:12]: `(?=\s)` would cover that use case as `\n` is part of `\s`. In eof the lookahead wouldn't match and thus `SELECT` would not be highlighted as keyword until a space or newline is added. With current solution SELECT is highlihted as keyword at the end of file.

Comment (jfcherng) [reply PHP/PHP Source.sublime-syntax]: Okay. I see. `\s` auto covers `$` for non-EOF situation.

**Review by deathaxe — APPROVED** [review]

**Review by Thom1729 — APPROVED** [review]

**Pull request merged**